### ISSUE TITLE
OGGM-hub info outdated

### ISSUE THREAD

**Issue by anoukvlug** [OGGM/oggm#1710]
[The information](https://docs.oggm.org/en/stable/cloud.html#oggm-hub) on how to get an OGGM-hub account seems outdated.

**Comment by fmaussion** [comment]
Yes - this should be moved out of the doc and added to a wiki, which is easier to update on short notice
